openfecwebapp/api_caller.py

```diff
@@ -88,9 +88,16 @@ def load_legal_advisory_opinion(ao_no):
 
     for document in documents:
         canonical_document = document
-        if (document['category'] == 'Final Opinion'):
+        if document['category'] == 'Final Opinion':
             break        
     
+    if not canonical_document:
+        return None
+
+    # manipulating category for display purposes
+    canonical_document['category'] = canonical_document['category'].lower()
+    canonical_document['category'] = canonical_document['category'].capitalize()
```

Comment:
Since these are display purposes, it doesn't make sense to change them at the data structure level. These should be done in the view.